Feature/bg design

Satyam1RP opened the pull request:
Background Design  Component Code is Done and ready for review.

mathrik commented:
I pulled down your branch, but the responsive behavior looked different than what you demo'd earlier today. Would you mind double checking your commits? I'd also appreciate it if you pasted the specs in a comment here.

Satyam1RP commented:
Hi @mathrik  here is the assignment file: 
[Assignment-09-07-2024.docx](https://github.com/user-attachments/files/16190171/Assignment-09-07-2024.docx)

I put the specifications from Assignment No. 8 to the scss file, and you may review all of them there as well.

Yes, the appearance differs from the demo I showed you; I updated the design and markup.
For this work, I set the first theme as the default and added three different versions of the same design, each with its own markup.

Required specs:  
![image](https://github.com/user-attachments/assets/4fd03e79-e47f-4687-997e-3ade5fdbfe6f)
 

here is the snapshot for mobile and desktop variation: 
Mobile: 
![image](https://github.com/user-attachments/assets/d9462b7a-c9e2-4aef-867f-3b9aa04f4815)

Desktop : 
![image](https://github.com/user-attachments/assets/6f5ff330-b67e-4f22-816c-bee6dac885cd)


CC: @jvaishnavrp @Lalit-gour1

jwetmore-hoodoo commented on Assets/scss/components/bg-design.scss:
```diff
@@ -0,0 +1,109 @@
+       
+    }
+
+    &-bottom{
```
Could potentially use an ::after pseudo-element instead, so you don't need to put the empty div in the HTML.

mathrik commented:
<img width="1111" alt="Screenshot 2024-07-12 at 3 21 07 PM" src="https://github.com/user-attachments/assets/3f28ede9-d7d5-4afb-b3b6-ee4512545890">
This is what I'm seeing when I build your code locally.

mathrik commented:
<img width="1111" alt="Screenshot 2024-07-12 at 3 21 07 PM" src="https://github.com/user-attachments/assets/3f28ede9-d7d5-4afb-b3b6-ee4512545890">
This is what I'm seeing when I build your code locally.

Satyam1RP commented:
@jwetmore-hoodoo Yes i used ::after now and removed blank div. 
@mathrik  can you take a look again I made a few changes on the last commit.

CC: @jvaishnavrp

jwetmore-hoodoo commented on Assets/scss/base/_var.scss:
```diff
@@ -27,8 +27,11 @@ $grey80: #808080;
 $darkestgrey: #352f2f;
 
 $wine: #3f2222;
+<<<<<<< HEAD
+=======
 $color-black-transparency: rgba(0, 0, 0, .8);
 $right-arrow: '\2192';
+>>>>>>> e78706feb29f743112caee8dcf4dc391612b9022
```
Git conflicts were committed.

Satyam1RP replied:
resolved.

mathrik commented:
<img width="959" alt="Screenshot 2024-07-15 at 10 19 22 AM" src="https://github.com/user-attachments/assets/0de30f54-5306-4783-852e-f1753b5fe819">
This is what I see today with the latest on this branch.

Satyam1RP commented:
hey @mathrik  I updated the code for the large screen can you retake the look?
